Dockerfile

```diff
@@ -52,9 +52,10 @@ RUN --mount=type=cache,target=/opt/ccache \
 FROM conda as conda-installs
 ARG PYTHON_VERSION=3.8
 ARG CUDA_VERSION=11.0
+ARG CUDA_CHANNEL=anaconda
```

Comment:
Shouldn't this be `defaults`?

I haven't actually tried using the `anaconda` channel before

Reply:
I found the `anaconda` channel in the Anaconda Cloud: https://anaconda.org/anaconda/cudatoolkit. But `conda config --show channels` gives `defaults`. I have updated to use the `defaults` channel.